sample/build.gradle

```diff
@@ -45,4 +45,5 @@ dependencies {
   }
   compile 'com.android.support:appcompat-v7:23.2.1'
   compile 'com.android.support:design:23.2.1'
+  compile 'com.mapzen.tangram:tangram:0.2-SNAPSHOT'
```

Comment:
I don't think we need this because we are transitively including dependencies from the library project